src/engine/specification.md

```diff
@@ -214,6 +214,7 @@ This structure contains the attributes required to initiate a payload build proc
 ##### Payload build process
 The payload build process is specified as follows:
 * Client software **MUST** set the payload field values according to the set of parameters passed into this method with exception of the `feeRecipient`. The built `ExecutionPayload` **MAY** deviate the `coinbase` field value from what is specified by the `feeRecipient` parameter.
+* Client software **MUST** reject `payloadAttributes` with timestamps that are more than one minute in the future from the clients current time or older or equal to the timestamp of the parent.
```

Comment:
Might consider defining "one minute" as a constant. Makes it easier to adjust and also draws more attention to the value for the reader

Reply:
I think the spec should be explicit about the outcome when the `payloadAttributes` gets rejected. Consider `engine_forkchoiceUpdated` returning an error in this case.